src/bijectors/pd.jl

```diff
@@ -26,12 +25,19 @@ function logabsdetjac(b::PDBijector, X::AbstractMatrix{<:Real})
     if !issuccess(Xcf)
         Xcf = cholesky(X + max(eps(T), eps(T) * norm(X)) * I)
     end
-    return logabsdetjac(b, Xcf)
+    return logabsdetjac_pdbijector_chol(Xcf)
 end
 
-function logabsdetjac(b::PDBijector, Xcf::Cholesky)
+function logabsdetjac_pdbijector_chol(Xcf::Cholesky)
+    # NOTE: Use `UpperTriangular` here because we only need `diag(U)`
+    # and `U` is by default already constructed in `Cholesky`.
```

Comment:
Not necessarily, could also be `L`.

Reply:
```suggestion
    # and `UL` is by default already constructed in `Cholesky`.
```

Reply:
By _default_ it's `U` no? But happy to change to `UL`; wasn't aware that was an option :+1: